march_joint_inertia_controller/include/march_joint_inertia_controller/joint_inertia_controller.h

```diff
@@ -3,23 +3,82 @@
 #ifndef MARCH_HARDWARE_JOINT_INERTIA_CONTROLLER_H
 #define MARCH_HARDWARE_JOINT_INERTIA_CONTROLLER_H
 
+#include <control_toolbox/pid.h>
 #include <controller_interface/controller.h>
-#include <hardware_interface/joint_command_interface.h>
-#include <pluginlib/class_list_macros.h>
+#include "march_joint_inertia_controller/inertia_estimator.h"
+#include <std_msgs/Float64.h>
 
 namespace joint_inertia_controller
 {
-class InertiaController : public controller_interface::Controller<hardware_interface::PositionJointInterface>
+class InertiaController : public controller_interface::Controller<hardware_interface::EffortJointInterface>
 {
 public:
+  struct Commands
+  {
+    double position_;    // Last commanded position
+    double velocity_;    // Last commanded velocity
+    bool has_velocity_;  // false if no velocity command has been specified
+  };
+
+  InertiaController();
```

Comment:
You have an empty constructor, so you can use `default` and remove the implementation
```suggestion
  InertiaController() = default;
```